test/unit/lib/urlutilTest.js

```diff
@@ -86,8 +86,11 @@ describe('urlutil', function () {
       it('is localhost (case-insensitive)', function () {
         assert.equal(UrlUtil.isNotURL('LoCaLhOsT'), false)
       })
-      it('ends with period (input contains a forward slash)', function () {
-        assert.equal(UrlUtil.isNotURL('https://brave.com/test/cc?_ri_=3vv-8-e.'), false)
+      it('ends with period (input contains a forward slash and domain)', function () {
+        assert.equal(UrlUtil.isNotURL('brave.com/test/cc?_ri_=3vv-8-e.'), false)
+      })
+      it('ends with period (input contains only a forward slash)', function () {
+        assert.equal(UrlUtil.isNotURL('brave/com/test/cc?_ri_=3vv-8-e.'), true)
```

Comment:
actually- I think this behavior might be OK!  I am almost positive you are correct about the other behavior (because it's happened to me so many times :stuck_out_tongue: ) When prefixed with protocol, it validates just fine: https://github.com/brave/browser-laptop/pull/6086/commits/cd88861cfa29f5e369001386b459ee0fe949cc57